src/main/java/com/blackducksoftware/integration/jira/task/maintenance/AlertMigrationRunner.java

```diff
@@ -59,32 +63,50 @@
 import com.blackducksoftware.integration.jira.web.model.JiraProject;
 import com.google.common.collect.ImmutableMap;
 
-public class AlertMigrationTask implements Callable<String> {
+public class AlertMigrationRunner implements JobRunner {
+    public static final String HUMAN_READABLE_TASK_NAME = "Black Duck Alert migration task";
     private static final String JIRA_QUERY_PARAM_NAME_ISSUE_TYPE = "issuetype";
     private static final String JIRA_QUERY_PARAM_NAME_PROEJCT = "project";
     private static final String JIRA_QUERY_CONJUNCTION = " AND ";
     private static final String JIRA_QUERY_DISJUNCTION = " OR ";
 
     private static final Integer MAX_BATCH_SIZE = 100;
+    private static final String NOT_STARTED_STATUS_MESSAGE = "Not started";
+    private static final String RUNNING_STATUS_MESSAGE = "Running";
     private static final String COMPLETE_STATUS_MESSAGE = "Complete";
-    private static final String NOT_CONFIGURED_STATUS_MESSAGE = "NOT CONFIGURED";
-    private static final String ERROR_STATUS_MESSAGE = "ERROR";
+    private static final String LOGGER_MESSAGE = "Please refer to the logs, make sure you have a logger configured for 'com.blackducksoftware.integration'.";
+    private static final String MIGRATION_NOT_NEEDED_MESSAGE = "If the plugin wasn't configured then there are no issues to migrate, please start using Alert instead";
 
     private final Logger logger = LoggerFactory.getLogger(getClass());
     private final JiraSettingsAccessor jiraSettingsAccessor;
     private final JiraServices jiraServices;
     private final IssuePropertyService issuePropertyService;
     private final MigrationAccessor migrationAccessor;
 
-    public AlertMigrationTask(JiraSettingsAccessor jiraSettingsAccessor, IssuePropertyService issuePropertyService) {
+    private Date startTime = null;
+    private Date endTime = null;
+    private String status = NOT_STARTED_STATUS_MESSAGE;
+
+    public AlertMigrationRunner(JiraSettingsAccessor jiraSettingsAccessor, IssuePropertyService issuePropertyService) {
         this.jiraSettingsAccessor = jiraSettingsAccessor;
         this.jiraServices = new JiraServices();
         this.issuePropertyService = issuePropertyService;
         this.migrationAccessor = new MigrationAccessor(jiraSettingsAccessor);
     }
 
     @Override
-    public String call() {
+    public JobRunnerResponse runJob(JobRunnerRequest jobRunnerRequest) {
+        this.startTime = jobRunnerRequest.getStartTime();
+        this.endTime = null;
+        this.status = RUNNING_STATUS_MESSAGE;
+        JobRunnerResponse jobRunnerResponse = runMigration();
+        this.status = jobRunnerResponse.getMessage();
+        this.startTime = null;
```

Comment:
Are we setting this to `null` because it was successful, so we want to display that in the UI?

Reply:
Good catch. I had removed this in another branch but hadn't updated this branch yet.